### DISCUSSION TITLE
4/12(수) 모나드

### DISCUSSION THREAD

**Discussion by Eunho0922** [Info-iOS/iOStudy#72] [General]
여태까지 모나드라는 문법은 하지 않은 것 같아 오늘의 과제로 만들었고, 모나드는 함수형 프로그래밍에서 사용되는 개념 중 하나로, 값을 포장하고 연산을 수행하는 방식입니다. 오늘 하루도 화이팅 입니당😀

**Comment by jjunhaa0211** [comment]
나보다 과제 잘 내는 듯 ㅋㅋ
